spec/requests/api/v1/countries/create_spec.rb

```diff
@@ -0,0 +1,47 @@
+describe 'POST /api/v1/countries', type: :request do
+  subject { post api_v1_countries_path, headers: headers, params: params, as: :json }
+
+  let(:headers) { auth_headers }
+  let(:user) { create(:user) }
+  let(:country_name) { Faker::Address.country }
+  let(:params) do
+    {
+      country: {
+        name: country_name
+      }
+    }
+  end
+
+  before { subject }
+
+  context 'when response is successful' do
+    it_behaves_like 'a successful request'
+
+    it 'returns country info' do
```

Comment:
Can we add another expectation for changing the countries count?
```ruby
it 'creates the country' do
  expect { subject }.to change { Country.count }.by(1)
end

Reply:
Updated